sklearn/manifold/t_sne.py

```diff
@@ -423,9 +423,11 @@ def trustworthiness(X, X_embedded, n_neighbors=5, precomputed=False):
         dist_X = X
     else:
         dist_X = pairwise_distances(X, squared=True)
-    dist_X_embedded = pairwise_distances(X_embedded, squared=True)
     ind_X = np.argsort(dist_X, axis=1)
-    ind_X_embedded = np.argsort(dist_X_embedded, axis=1)[:, 1:n_neighbors + 1]
+    neigh_embedded = NearestNeighbors(n_neighbors).fit(X_embedded)
+    ind_X_embedded = neigh_embedded.kneighbors(
+        None, n_neighbors, return_distance=False
```

Comment:
No need to repeat n_neighbors here. Fit on one line?

Reply:
`Fit on one line?` you mean call `kneighbors` directly after fit without storing the `neigh_embedded ` var?